.lando.yml

```diff
@@ -44,6 +44,17 @@ services:
         interval: 5s
         timeout: 3s
         retries: 3
+  trifid:
```

Comment:
What do you use trifid for?

Reply:
Just for YASGUI. It's quite annoying that Fuseki's UI only has a single query. Multiple tabs are useful